imod/mf6/model.py

```diff
@@ -495,17 +495,21 @@ def regrid_like(
                 raise ValidationError("\n" + status_info.to_string())
         return new_model
 
-    def _mask_all_packages(
+    def mask_all_packages(
         self,
-        domain: GridDataArray,
+        mask: GridDataArray,
     ):
         """
         This function applies a mask to all packages in a model. The mask must
         be presented as an idomain-like integer array that has 0 or negative
```

Comment:
You mention "negative" values and "positive", have you tested if masking also works for -2 and 2? Otherwise just be more specific: mention -1 and 1.